base/ca/shared/conf/CS.cfg

```diff
@@ -957,23 +957,23 @@ log.instance.Transactions.rolloverInterval=2592000
 log.instance.Transactions.type=transaction
 logAudit.fileName=[PKI_INSTANCE_PATH]/logs/[PKI_SUBSYSTEM_TYPE]/access
 logError.fileName=[PKI_INSTANCE_PATH]/logs/[PKI_SUBSYSTEM_TYPE]/error
-oidmap.auth_info_access.class=netscape.security.extensions.AuthInfoAccessExtension
+oidmap.auth_info_access.class=org.mozilla.jss.netscape.security.extensions.AuthInfoAccessExtension
```

Comment:
Assuming the old classes will no longer exist, we need to provide an upgrade script so that existing instances will continue to work.

Reply:
Ahh right, these are configurations.

#122 removes the old classes, so yes, I'll try creating an upgrade script.